GVRf/Framework/framework/src/main/java/org/gearvrf/GVRViewManager.java

```diff
@@ -688,7 +690,9 @@ protected void captureCenterEye(GVRRenderTarget renderTarget, boolean isMultivie
         // TODO: when we will use multithreading, create new camera using centercamera as we are adding posteffects into it
         final GVRCamera centerCamera = mMainScene.getMainCameraRig().getCenterCamera();
         final GVRMaterial postEffect = new GVRMaterial(this, GVRMaterial.GVRShaderType.VerticalFlip.ID);
-        centerCamera.addPostEffect(postEffect);
+
+        if(!(isVulkanInstance() && mBackend == 6))
```

Comment:
This is a problem. Please look into moving these lines to the monoscopic view manager; you could override captureCenterEye there.

Reply:
And lets try to figure out a way that doesn't require isVulkanInstance, is possible. And if it is not possible, let's open an issue to track what I presume is a problem with the design.

Reply:
Have removed isVulkanInstance and handling the monoscopic cases in the native Vulkan side.